sdk/maps/tests.yml

```diff
@@ -7,4 +7,7 @@ extends:
     BuildInParallel: true
     TimeoutInMinutes: 180
     Clouds: 'Public'
-    SupportedClouds: 'Public,UsGov,Canary'
\ No newline at end of file
+    SupportedClouds: 'Public,UsGov,Canary'
+    EnvVars:
+      AZURE_MAPS_TEST_MODE: 'Live'
```

Comment:
Any reason we need another specific environment variable for maps here? We already have a common variable for it, https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/jobs/live.tests.yml#L141.

Reply:
Thanks, I have removed the unnecessary environment variable